images/manageiq-base/container-assets/container_env

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+update-ca-trust
```

Comment:
What provides this command?  If it's mod_ssl, and the change to add mod_ssl is only in the ui worker image, then the other images will fail.

Reply:
It's part of `ca-certificates-2020.2.41-80.0.el8_2.noarch` which comes with UBI